template/android/app/src/development/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">BaseDev</string>
+    <string moduleConfig="true" name="CodePushDeploymentKey">DeploymentKey</string>
```

Comment:
Could you try to config it using env file.

Reply:
i've updated